tasks.py

```diff
@@ -162,3 +165,31 @@ def type_check(ctx, packages, install_types=False, show_default_packages=False):
     if install_types:
         cmds.extend(["--install-types", "--non-interactive"])
     ctx.run(" ".join(cmds), echo=True)
+
+
+@invoke.task(aliases=["get-stats"])
+def get_usage_stats_json(ctx):
+    """
+    Dump usage stats event examples to json file
+    """
+    events = usage_stats_utils.get_usage_stats_example_events()
+    version = usage_stats_utils.get_gx_version()
+
+    outfile = f"v{version}_example_events.json"
+    with open(outfile, "w") as f:
+        json.dump(events, f)
+
+    print(f"File written to '{outfile}'.")
+
+
+@invoke.task(get_usage_stats_json, aliases=["move-stats"])
```

Comment:
```suggestion
@invoke.task(pre=[get_usage_stats_json], aliases=["move-stats"])
```

What you already have works fine, but this is slightly more expressive.
https://docs.pyinvoke.org/en/0.11.0/concepts/execution.html#pre-and-post-tasks